doc/ansible/user-guide.md

```diff
@@ -256,23 +256,37 @@ Once this is done, there are two ways to run the operator:
 
 Running as a pod inside a Kubernetes cluster is preferred for production use.
 
+Setup the DOCKER_IMAGE environment variable corresponding to your quay registry:
+```
+$ export QUAY_USERNAME="put your quay username here"
+$ export DOCKER_IMAGE="quay.io/${QUAY_USERNAME}/memcached-operator:v0.0.1"
+```
```

Comment:
IHMO would be great keep it equals across the repo.
Note that these are the same steps for GO and Helm as well.

Reply:
I am a bit confused;  are you saying:
1) You would rather I not make this change, since it conflicts with the established style.
or
2) You are fine with this change, but want me to change the verbage in the GO and Helm sections to match the steps I introduced in this change.

Thanks!

Reply:
@camilamacedo86 Maybe one option for this long-term (since we're switching to a hugo/docsy-based website for docs) would be to create a [partial template](https://gohugo.io/templates/partials/) for this section and then include it into each of the guides so that it is consistent everywhere, but doesn't force those reading the docs to click through different links.

@ryandgoulding IMO for right now, I think option 2 would be preferred.

Reply:
Yes. The template would be great. But, now we could move with option 2. 👍